http/http.go

```diff
@@ -67,29 +76,31 @@ func (m *Module) Struct() *starlarkstruct.Struct {
 // StringDict returns all module methods in a starlark.StringDict
 func (m *Module) StringDict() starlark.StringDict {
 	return starlark.StringDict{
-		"get":     starlark.NewBuiltin("get", m.reqMethod("get")),
-		"put":     starlark.NewBuiltin("put", m.reqMethod("put")),
-		"post":    starlark.NewBuiltin("post", m.reqMethod("post")),
-		"delete":  starlark.NewBuiltin("delete", m.reqMethod("delete")),
-		"patch":   starlark.NewBuiltin("patch", m.reqMethod("patch")),
-		"options": starlark.NewBuiltin("options", m.reqMethod("options")),
+		"get":      starlark.NewBuiltin("get", m.reqMethod("get")),
+		"put":      starlark.NewBuiltin("put", m.reqMethod("put")),
+		"post":     starlark.NewBuiltin("post", m.reqMethod("post")),
+		"postForm": starlark.NewBuiltin("postForm", m.reqMethod("postForm")),
```

Comment:
into it. I'm a fan of go's `PostForm`

Reply:
Oops. This is actually left over from a different implementation approach. The `postForm` functionality is handled by setting `form_encoding`. I'll remove this `postForm` thing.

Reply:
perf. Thanks!